src/index.ts

```diff
@@ -699,3 +702,88 @@ const getAvailableOn = (serp: Serp, $: CheerioStatic) => {
     serp.availableOn = availableOn;
   }
 };
+
+const getShopCards = (serp: Serp, $: CheerioStatic) => {
+  const CONFIG = {
+    currency: '.e10twf',
+    imgLink: 'a.pla-unit-img-container-link',
+    price: '.e10twf',
+    shopFeature: '.top-pla-group-inner',
+    shopOffer: '.pla-unit:not(.view-all-unit)',
+    shoppingSite: '.LbUacb',
+    title: 'a > .pymv4e',
+  };
+  const shopFeature = $(CONFIG.shopFeature);
+  if (shopFeature.length) {
+    const shop: ShopCard[] = [];
```

Comment:
@LukaGrdinic use plural for lists, shops instead of the shop
```suggestion
    const shops: ShopCard[] = [];
```
A bit more into it, the name itself is wrong more about this on models.ts file